OSVectorTileAPI/StylingVTS/Adapting/README.md

```diff
@@ -0,0 +1,33 @@
+# Adapting the pre-defined style
+
+See the "Adapting" folder for a completed example.

```

Comment:
This readme is already in that folder, so the text should probably read:
See the html and javascript in this folder for a completed example.
The same comment applies to the ExternalFile and InCode examples

Reply:
Done. In next commit.